lib/vbox/vbox.go

```diff
@@ -53,8 +53,18 @@ type (
 		Url  string `json:"url"`
 		Port string `json:"url"`
 	}
+
+	OnOff bool
 )
 
+func (o OnOff) String() string {
+	if o {
+		return "on"
+	} else {
```

Comment:
__[golint]__
* [golint] if block ends with a return statement, so drop this else and outdent its block

<!-- Comment by [SideCI](https://sideci.com) -->